ui/packages/lib/src/components/page_placeholder/PagePlaceholder.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingContent,
+  EuiPageTemplate
+} from "@elastic/eui";
+
+export const PagePlaceholder = () => (
+  <EuiPageTemplate restrictWidth="90%">
```

Comment:
Are we not moving this to a global setting? 😄

Reply:
I'm planning to do so in one of the following MRs 🤫